pkg/lib/cockpit-components-file-autocomplete.jsx

```diff
@@ -101,21 +111,10 @@ export class FileAutoComplete extends React.Component {
         if (value && value.indexOf("/") !== 0)
             value = "/" + value;
 
-        if (this.timer)
-            window.clearTimeout(this.timer);
-
         if (this.state.value !== value)
-            this.timer = window.setTimeout(() => {
-                this.timer = null;
-
-                if (!this.updateIfDirectoryChanged(value)) {
-                    var stateUpdate = this.filterFiles(value);
-                    this.setState(stateUpdate);
-                    this.onChangeCallback(value, { error: stateUpdate.error });
-                }
-            }, 250);
-
+            this.throttledChange(value);
         this.setState({ value });
+        this.onChangeCallback(value, {});
```

Comment:
Previously, the `onChange` property callback wouldn't have been called for every key stroke/change, but only in the "debounced" timer handler above. This will now call it on every change. As long as the handler is called eventually, I think it makes sense to not increase the calling frequency unnecessary. Or was this a bug fix for c-podman?